resources/views/home/_changelog_stream.blade.php

```diff
@@ -0,0 +1,30 @@
+{{--
+    Copyright 2015-2017 ppy Pty. Ltd.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.i
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+<a
+  class="changelog-header__stream {{ $featured ? 'changelog-stream--featured' : '' }} changelog-stream {{ $streamId === $stream->stream_id ? 'changelog-stream--active' : '' }} changelog-stream--{{ str_slug($stream->updateStream->pretty_name) }}"
```

Comment:
avoid mixing block and element. Also avoid having element with the block in different file.